client/overview/style.scss

```diff
@@ -13,7 +13,17 @@
 	position: relative;
 
 	.woocommerce-experimental-list__item {
-		position: initial;
+		position: relative;
```

Comment:
This fixed a styling bug where the border-left of one task was overflowing across multiple tasks.

Reply:
I don't know how to reproduce this. Do you have screenshots?

Reply:
This can be reproduced by enabled other tasks in the list: force this by returning `true` for `showUpdateDetailsTask` and `wpcomReconnectUrl` in [WC_Payments_Admin::get_js_settings()](https://github.com/Automattic/woocommerce-payments/blob/5a3909d4823d3794f53f73fed5d468ca905ca656/includes/admin/class-wc-payments-admin.php#L763-L764).

**With `position: initial` (before this PR)**
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/d6726d9f-7d8f-4882-8d67-7cf8edf0c1ae)

**With `position: relative` (after this PR)**

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/4fe7ed11-7645-47bd-b4f1-ac7eacc025a0)

Reply:
Thanks for explaining that.